transmit-lib/src/cmr/transmit/http_helper.clj

```diff
@@ -141,6 +141,39 @@
                                          :accept :json}
                                         http-options#)}))))))
 
+(defmacro defcreator2
```

Comment:
Does a second have to be made or can the original be modified? And if not does this supersede `defcreator`? If yes `defcreator` needs to be marked as deprecated. If it does not replace defcreateor, we shouldn't just serialize the number. `defcreator-with-opts` or `defcreator-from-json`

Reply:
We don't need the defcreator anymore.  I created issue https://bugs.earthdata.nasa.gov/browse/CMR-8622 to remove it.